src/views/CharacterListView.js

```diff
@@ -1,35 +1,46 @@
-import React from "react";
-import { connect } from "react-redux";
+import React from 'react';
+import { connect } from 'react-redux';
 
-import { CharacterList } from "../components";
+import { CharacterList } from '../components';
 // import actions
+import { getSWChars } from '../actions';
 
 class CharacterListView extends React.Component {
-  constructor() {
-    super();
-  }
-
   componentDidMount() {
     // call our action
+    this.props.getSWChars();
   }
 
   render() {
     if (this.props.fetching) {
       // return something here to indicate that you are fetching data
+      return (
+        <h3>"That's no moon. It's a...loading screen." - Obi-Wan Kenobi</h3>
+      );
     }
     return (
-      <div className="CharactersList_wrapper">
-        <CharacterList characters={this.props.characters} />
+      <div className='CharactersList_wrapper'>
+        <CharacterList
+          characters={this.props.characters}
+          error={this.props.error}
+        />
       </div>
     );
   }
 }
 
 // our mapStateToProps needs to have two properties inherited from state
 // the characters and the fetching boolean
+const mapStateToProps = state => ({
+  characters: state.characters,
+  fetching: state.fetching,
+  error: state.error
```

Comment:
here's another way your syntax in combineReducers causes some issues: you need to reference `state.characters` to be referencing the actual state object being returned by the reducer, but since you replace the charsReducer reference with just `characters`, you now have to do it for all of them: if you console.log your props in this component, you'll see that your fetching and error props are undefined. if you leave your combineReducer the way it is, you would need to say `fetching: state.characters.fetching` and `error: state.characters.error` which is counterintuitive syntax, suggesting that the actual characters list is what holds the fetching and error values. This is another reason why I'd leave the combineReducer to just return and object with `charsReducer`, because then you can semantically reference the values with `state.charsReducer.characters/fetching/error` -- or take it one step further and destructure the state being passed in to your mSTP however you'd like. This would be important if you had multiple reducers going into your combineReducer, as you want to reference them by name specifically here